### PR TITLE
jira/HZN-332

### PR DESCRIPTION
**Pull request by christianpape**
Please review and merge:

http://issues.opennms.org/browse/HZN-332

### CODE REVIEW

File: opennms-webapp/src/main/webapp/index.jsp [hunk 1]
```diff
@@ -52,8 +52,20 @@
 
 	<!-- Middle Column -->
 	<div class="col-md-6" id="index-contentmiddle">
-		<% String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp"); %>
-		<jsp:include page="<%=centerUrl%>" flush="false" />
+		<%
+			String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp");
+
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please remove empty line

File: opennms-webapp/src/main/webapp/index.jsp [hunk 2]
```diff
@@ -52,8 +52,20 @@
...
+			String[] centerUrlArr = centerUrl.split(",");
+
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please remove empty line

File: opennms-webapp/src/main/webapp/index.jsp [hunk 3]
```diff
@@ -52,8 +52,20 @@
...
+			for(String centerUrlItem : centerUrlArr) {
+		%>
+
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please remove empty line

File: opennms-webapp/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -52,8 +52,20 @@
...
+		<jsp:include page="<%=centerUrlItem%>" flush="false" />
+
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please remove empty line

File: opennms-webapp/src/main/webapp/index.jsp [hunk 5]
```diff
@@ -52,8 +52,20 @@
...
+		<%
+			}
+		%>
+
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please remove empty line

File: opennms-webapp/src/main/webapp/index.jsp [hunk 6]
```diff
@@ -52,8 +52,20 @@
...
-		<jsp:include page="<%=centerUrl%>" flush="false" />
+		<%
+			String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp");
+
+			String[] centerUrlArr = centerUrl.split(",");
+
+			for(String centerUrlItem : centerUrlArr) {
+		%>
+
+		<jsp:include page="<%=centerUrlItem%>" flush="false" />
```
Comment (indigo423) [inline opennms-webapp/src/main/webapp/index.jsp]: please :) indentation as a member of the for statement, makes the code easier to read.

**Comment by mvrueden** [comment]
Hey Ulf, can you please recheck christianpape's OCA?

**Pull request merged**